tests/commands_test.py

```diff
@@ -169,6 +169,10 @@ def test_broken_plugin(testbot):
         )
         assert "import borken  # fails" in testbot.pop_message()
         assert "as it did not load correctly." in testbot.pop_message()
+        assert (
+            "Error: Broken failed to activate: "
+            "'NoneType' object has no attribute 'is_activated'"
+        ) in testbot.pop_message()
```

Comment:
This new error happens because this line returns `None`:

https://github.com/browniebroke/errbot/blob/c81c78ec29592cb17d29f3aaa4b954f23f9a77ce/errbot/plugin_manager.py#L431

Which cause an attribute error down here:

https://github.com/browniebroke/errbot/blob/c81c78ec29592cb17d29f3aaa4b954f23f9a77ce/errbot/plugin_manager.py#L439

Open to suggestions on how to fix it better.